app/src/components/user_details_form.rs

```diff
@@ -115,6 +122,45 @@ impl Component for UserDetailsForm {
               <StaticValue label="User ID" id="userId">
                 <i>{&self.user.id}</i>
               </StaticValue>
+              {
```

Comment:
It seems like a much cleaner way of doing this would be to add the field to the hardcoded attributes. Then it would be handled by the existing loop below. You basically wouldn't have to touch this file at all